ParisBNF/abb/BNFabb3.xml

```diff
@@ -0,0 +1,327 @@
+                        <seg type="ink">Black, red</seg>
+                        </handNote>
+                  </handDesc>
+
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus target="#128r"/>
+                           <desc type="Unclear">
+                             A note related to tributes,
+                             possibly in the region of <placeName ref="LOC1572Asgade"/>.</desc>
+                          <q xml:lang="am">
+                            ከ<placeName ref="LOC1572Asgade">አስገዴ፡</placeName> ፭፻፵፱ለሸለቃ፡ ሽቴ፡ ፫፻፺፭ለአዕበዮ፡ አራ፡
+                            ፬፻፸፩ለገብራይ፡ አድአቀይት፡ ፬፻፹፫ለአሰላፊ፡ ወልደ፡ ሚካኤል፡ <sic resp="DN">፫፻፺፮ለወድ፡</sic> ሸለቃ፡
+                             ፍሥሐ፡ ፫፻፷፪ለተስፋይ፡ ግባ፡ ደግ፡ <sic resp="DN">፬፻ለወድ፡</sic> ሸለቃ፡ ፍሥሐ፡ ገብረ፡ ሚካኤል።
+                              </q>
+                              <note>French translation in <bibl><ptr target="bm:ContiRossini1914Notice"/>
+                            <citedRange unit="page">49</citedRange></bibl>; words <foreign>waddi</foreign>
+                          and <foreign>ʾasallāfi</foreign> are noted as Tǝgrǝñña forms.</note>
+                        </item>
+
+                        <item xml:id="a2">
+                           <locus target="#129r"/>
+                           <desc type="Unclear">A note concerning the <foreign xml:lang="gez">rim</foreign>-land of a
+                            a church, from the time of the king <persName ref="PRS9142TaklaGi">Takla Giyorgis</persName></desc>
+                           <q xml:lang="gez">
+                             <persName ref="PRS13326GabraMaryam">አበ፡ ገብረ፡ ማርያም፡<sic resp="CR"/></persName> የወንደባሪ፡ ልጅ፡ አገሩ፡ <placeName ref="LOC1413Ambass">አምባሴል፡</placeName>
+                            <placeName ref="LOC1390AmbaGe">ግሼን፡</placeName>
+                             ጉልቱ፡ አስፋ፡ ገዚ፡ የናቱ፡ ቤት፡ አገሩዋ፡ <placeName ref="LOC6287Wecale">ዎጫሊ፡
+                           </placeName> ምላቼ፡ አባቱዋን፡ አቶ፡ ሞቲ፡ <sic resp="CR">እርሶዋን፡</sic> መነን፡ <sic resp="CR">ወንድሞዎ፡</sic> <persName ref="PRS13325Adilu">አዲሉ</persName>።
+                              </q>
+                              <note>French translation in <bibl><ptr target="bm:ContiRossini1914Notice"/>
+                            <citedRange unit="page">49</citedRange></bibl>.</note>
+                        </item>
+
+                        <item xml:id="a3">
+                           <locus target="#129r"/>
+                           <desc type="Unclear">A note added by <persName ref="PRS1071dAbbadi">Antoine dʼAbbadie
+                           </persName>, referring to one of the two
+                           mentioned in <ref target="#a2">Addition 2</ref>,
+                           most probably to <persName ref="PRS13325Adilu">ʾAdilu</persName>.</desc>
+                           <q xml:lang="fr">
+                            Cʼest le nom de celui qui me lʼa donnè (le manuscrit); à la guerre
```

Comment:
donné?

Reply:
Interesting information on the name, maybe it could be included in the person record? (nom de guerre? But I am no expert)

Reply:
donné, indeed. Abbadie says that it is nom de guerre, but, if it is, it is a strange one. Anyway, how can I encode "nom de querre"?

Reply:
persName type="war" in the person record

Reply:
ok, done in the personal record.